manifests/init.pp

```diff
@@ -151,6 +157,17 @@
       key     => $certs::apache::apache_key,
       ca_cert => $certs::ca_cert,
       require => Class['certs::apache'],
+      port    => $docker_registry_port,
+    }
+  }
+
+  if $pulp_master {
+    $check_file = file('/usr/sbin/foreman-rake', '/dev/null')
+    if $check_file != '' {
+      foreman_config_entry { 'pulp_docker_registry_port':
```

Comment:
I suspect you are correct. This is what I get on a forklift w/ katello.yml playbook:
```
[ERROR 2016-10-26 09:29:12 main]  Execution of '/usr/sbin/foreman-rake -- config -k 'pulp_docker_registry_port' -v '5001'' returned 1: rake aborted!
[ERROR 2016-10-26 09:29:12 main] Gem::LoadError: Specified 'sqlite3' for database adapter, but the gem is not loaded. Add `gem 'sqlite3'` to your Gemfile (and ensure its version is at the minimum required by ActiveRecord).
```

I am going to need assistance in getting this to work. I'll continue to seek guidance on IRC.